### ISSUE TITLE
defineMessages isn't parsed when saved in a ts file build with parcel-builder

### ISSUE THREAD

**Issue by smeijer** [akameco/babel-plugin-react-intl-auto#79]
* version: 2.3.0
* `node` version: 8.16.1
* `npm` (or `yarn`) version: 6.11.3

**Do you want to request a _feature_ or report a _bug_?:**
BUG

**What is the current behavior?:**
When running a project that makes use of this plugin with [`parcel`](https://parceljs.org), it throws the error:

```
Uncaught Invariant Violation: [React Intl] An `id` must be provided to format a message.
```

When debugging, the `messages` still show as a string:string dictionary. The strings are not being converted into a message descriptor. So it looks like this plugin isn't being applied.

**What is the expected behavior?:**
Having transpiled message descriptors.

**Comment by akameco** [comment]
@smeijer  Please show the Reproduction repository.

**Comment by smeijer** [comment]
@akameco, I was able to reproduce in codesandbox. The link is below.

I also traced the error down. You'll find a file `messages.ts` in the `src` folder. As soon as that one is renamed to `messages.js`, the error is gone. So it seem to have something to do with the typescript transpiler.

https://codesandbox.io/s/react-parcel-starter-jk795

![react-intl-auto](https://user-images.githubusercontent.com/1196524/66712767-e2acbe00-eda1-11e9-9036-b6095643e854.gif)
